SU-SWS/stanford_paragraph_types
# EARTH-298: Added section callout cards and callout card paragraph types.

# READY FOR REVIEW

# Summary
- Enables the callout cards and callout card paragraph types.

# Needed By (Date)
- Friday the 21st please.

# Criticality 
- Content authors must haz. Need before July release.

# Steps to Test

1.  Check out EARTH-298 in each of stanford-earth/SE3_BLT, SU-SWS/stanford_components, and SU-SWS/stanford_paragraph_types
2. Import the config `blt sws:config-import` or the usual drush commands if that doesn't work
3. Clear caches and edit a complex page
4. Add a new **Callout Cards** paragraph type and a few cards
5. Review looks and feels.

# Associated Issues and/or People
- EARTH-298

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
